sklearn/utils/seq_dataset.pyx.tp

```diff
@@ -1,10 +1,44 @@
+{{py:
+
+"""
+Dataset abstractions for sequential data access.
+Template file for easily generate fused types consistent code using Tempita
+(https://github.com/cython/cython/blob/master/Cython/Tempita/_tempita.py).
+
+Generated file: seq_dataset.pyx
+
+Each class is duplicated for all dtypes (float and double). The keywords
+between double braces are substituted in setup.py.
+
+Author: Peter Prettenhofer <peter.prettenhofer@gmail.com>
+        Arthur Imbert <arthurimbert05@gmail.com>
+        Joan Massich <mailsik@gmail.com>
+
+License: BSD 3 clause
+"""
+
+# name, c_type, np_type
+dtypes = [('64', 'double', 'np.float64'),
+          ('32', 'float', 'np.float32')]
+
+def get_dispatch(dtypes):
+    for name, c_type, np_type in dtypes:
+        yield name, c_type, np_type
+
+}}
+
+{{for name, c_type, np_type in get_dispatch(dtypes)}}
+
+#------------------------------------------------------------------------------
+
+"""
+Dataset abstractions for sequential data access.
+WARNING: Do not edit .pyx file directly, it is generated from .pyx.tp
+"""
+
 # cython: cdivision=True
 # cython: boundscheck=False
 # cython: wraparound=False
-#
-# Author: Peter Prettenhofer <peter.prettenhofer@gmail.com>
-#
-# License: BSD 3 clause
```

Comment:
> We should probably keep this part and repeat it in both template file

https://github.com/scikit-learn/scikit-learn/pull/11155/files#r203940547

Reply:
@glemaitre 	I'm not sure about that, its alredy in he header and by the end this is a compiled `seq_dataset.pyx`